starlite/datastructures/response_containers.py

```diff
@@ -57,6 +59,8 @@ class Config(BaseConfig):
     """A list of Cookie instances to be set under the response 'Set-Cookie' header. Defaults to None."""
     media_type: Optional[Union[MediaType, str]] = None
     """If defined, overrides the media type configured in the route decorator"""
+    encoding: str = "utf-8"
+    """The encoding to used for the response headers."""
```

Comment:
Probably this should be `The encoding to be used for the response headers`

Reply:
fixed